storybook/src/elements/nuxeo-data-list/nuxeo-data-list.stories.js

```diff
@@ -0,0 +1,86 @@
+import { html } from 'lit-html';
+import { storiesOf } from '@storybook/polymer';
+import { analyse } from '../../../.storybook/analysis';
+import { LIST, LIST_EMPTY } from '../../data/lists.data.js';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-data-list/nuxeo-data-list.js';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-document-thumbnail/nuxeo-document-thumbnail.js';
+
+const docs = analyse('nuxeo-data-list').notes;
+const stories = storiesOf('UI/nuxeo-data-list', module);
+const _computedClass = (isSelected) => {
+  let classes = 'list-item';
+  if (isSelected) {
+    classes += ' selected';
+  }
+  return classes;
+};
+
+stories
+  .add(
+    'Empty',
+    () =>
+      html`
+        <style>
+          * {
+            font-family: 'Open Sans', Arial, sans-serif;
+          }
+        </style>
+        <nuxeo-data-list .items="${LIST_EMPTY}"></nuxeo-data-list>
+      `,
+    { notes: { markdown: docs } },
+  )
+  .add(
+    'Default',
+    () =>
+      html`
+        <style>
+          * {
+            font-family: 'Open Sans', Arial, sans-serif;
+          }
+          .list-item {
+            display: flex;
+            justify-content: left;
+            align-items: center;
+          }
+          nuxeo-document-thumbnail {
+            display: block;
+          }
+        </style>
+        <nuxeo-data-list .items="${LIST.data}">
+          <template>
+            <div tabindex$="{{tabIndex}}" class="${_computedClass()}">
```

Comment:
This is not a polymer binding. This will be run once per story and not per item, as one would expect.